test/data/testvib.py

```diff
@@ -222,6 +225,18 @@ class QChemRamanTest(GenericRamanTest):
     max_raman_intensity = 588
 
 
+class GenericLinearIRTest(GenericIRTest):

```

Comment:
Related to https://github.com/cclib/cclib/issues/426#issuecomment-324831305, do we want this as a unit test or as part of regressions?